apps/backend/app/services/job_service.py

```
@@ -12,125 +14,364 @@
+            
+            return result
+            
+        except ValidationError as e:
+            logger.error(f"Validation error in job extraction: {e}")
+            # Return partial result with extracted keywords
+            return {
+                "job_title": self._extract_job_title(job_description_text),
+                "extracted_keywords": [kw[0] for kw in keywords],
+                "job_summary": job_description_text[:500]
+            }
+        except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)